HyperVExtension/src/HyperVExtension/CommunicationWithGuest/WmiUtility.cs

```diff
@@ -124,4 +138,47 @@ public static bool JobCompleted(ManagementBaseObject outParams, ManagementScope
 
         return jobCompleted;
     }
+
+    public static FeatureAvailabilityKind GetHyperVFeatureAvailability()

```

Comment:
Why have a check in two places, the extension and Dev Home? Won't one be enough?

Reply:
(Haven't looked at the detail but) generally the extension and the core should not be calling into each other outside of the SDK. Pretend like the extension was in its own repo.

Reply:
I decided to add it to the extension just for logging purposes. It would be good for the extensions logs to show this information as well. It would be nice if I only needed to have it in one place but I can't share Dev Home's project with the extension.